src/Action.hs

```diff
@@ -104,12 +110,6 @@ instance PersistM r => ActionPersist r (Action r) where
 instance MonadIO (Action r) where
     liftIO = Action . liftIO
 
-instance PersistM r => PersistM (Action r) where
```

Comment:
This instance should stay. @np idea is to remove all the non necessary `persistent` calls from the codebase. I want to do that, after this PR is merged. Please let this instance live. :)